label_studio/tests/test_contextlog.py

```diff
@@ -0,0 +1,25 @@
+"""This file and its contents are licensed under the Apache License 2.0. Please see the
+included NOTICE for copyright information and LICENSE for a copy of the license.
+"""
```

Comment:
Ah I was actually proposing we remove the license string, I don't think these are needed